ui/components/ui/site-origin/site-origin.js

```diff
@@ -1,16 +1,35 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import classnames from 'classnames';
 import Chip from '../chip';
 import IconWithFallback from '../icon-with-fallback';
 
-export default function SiteOrigin({ siteOrigin, iconSrc, iconName }) {
+export default function SiteOrigin({
+  siteOrigin,
```

Comment:
`siteOrigin` is used in both rendering cases, so I think we should make it `required` in `props` validation.

Reply:
🤔  I think it is already required? [line 38](https://github.com/MetaMask/metamask-extension/pull/14720/files#diff-518115d24597fde0bd8b5e57f626086ac81d13595fc3b90335e2972ae76e654cR38)

Reply:
d'oh, sorry for not seeing that!